.gitignore

```diff
@@ -10,3 +10,6 @@ __pycache__/
 
 # App system files
 /network_manager.egg-info/
+
+# database
+*.sqlite
```

Comment:
we want to allow the user to decide which database backend they want. If they choose sqlite then we don't want it to be accidentally committed to the repo

Reply:
I see.  So this essentially keeps one database type to be used exclusively for development or personal use